src/Features/CSharp/Portable/CSharpFeaturesResources.resx

```diff
@@ -523,9 +523,6 @@
   <data name="Convert_to_switch_expression" xml:space="preserve">
     <value>Convert to 'switch' expression</value>
   </data>
-  <data name="Warning_Extracting_a_local_function_reference_may_produce_invalid_code" xml:space="preserve">
```

Comment:
Why isn't this needed anymore? What about the normal Extract Method case?

Reply:
Nevermind. Because you block Extract Method in this case. Seems reasonable to me.